docs(login examples): Path Fixes for  login and firebase login examples (#3227)
felangel/bloc: docs/flutterlogintutorial.md

- [BlocProvider](/flutterbloccoreconcepts?id=blocprovider), Flutter widget which provides a bloc to its children.
- [BlocBuilder](/flutterbloccoreconcepts?id=blocbuilder), Flutter widget that handles building the widget in response to new states.

> - Using Bloc instead of Cubit. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)

Reviewer: ```suggestion
- Using Cubit and Bloc. [What's the difference?](/coreconcepts?id=cubit-vs-bloc)
```

flutter create flutter_login

> ```

Reviewer: ```suggestion
```

Next, we can install all of our dependencies

```sh
flutter packages get
```
```

> ## Authentication Repository

Reviewer: ```suggestion
Next, we can install all of our dependencies

```sh
flutter packages get
```

## Authentication Repository
```

The first thing we're going to do is create an `authentication_repository` package which will be responsible for managing the authentication domain.

> We'll start by creating `packages/authentication_repository` directory at the root of the project which will contain all internal packages.

Reviewer: ```suggestion
We'll start by creating a `packages/authentication_repository` directory at the root of the project which will contain all internal packages.
```

```sh

> flutter packages get

Reviewer: ```suggestion
flutter packages get
```

## Authentication Repository
```

We can install the dependencies by running:

```sh

> flutter pub get

Reviewer: flutter packages get should be fine as well.

> flutter pub get

Reviewer: ```suggestion
flutter packages get
```
